oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/reactive/function/client/ServletOAuth2AuthorizedClientExchangeFilterFunction.java

```diff
@@ -100,8 +93,10 @@
  * @since 5.1
  * @see OAuth2AuthorizedClientManager
  */
-public final class ServletOAuth2AuthorizedClientExchangeFilterFunction
-		implements ExchangeFilterFunction, InitializingBean, DisposableBean {
+public final class ServletOAuth2AuthorizedClientExchangeFilterFunction implements ExchangeFilterFunction {
```

Comment:
@rwinch What are your thoughts on removing `InitializingBean` and `DisposableBean`? Just a reminder that this was backported to 5.1.x via this [commit](https://github.com/spring-projects/spring-security/commit/091cbe54585b44b5656cbb33a6d2e29a82e7f05f#diff-972147f7b036129a221ef5e0a9da5886R108). In hindsight, it probably should not have since it introduced new API's in a patch release. So I feel removing it here is fine but want to get your take on this. If you prefer I leave it in than I can simply do no-op in both methods.

Reply:
I would just make them noops

Reply:
Actually in hindsight we are already breaking compatibility because for the functionality to work the same way `SecurityReactorContextSubscriberRegistrar` needs to be added. I'd go ahead and remove it so it is more apparent.

Reply:
Ok so I'll leave the PR as-is, which has already removed `InitializingBean` and `DisposableBean`